src/SFA.DAS.AANHub.Application/Employers/Queries/GetEmployerMemberResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Employers.Queries
+{
+    public class GetEmployerMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string Email { get; set; } = null!;
+        public string Name { get; set; } = null!;
+        public string Organisation { get; set; } = null!;
+        public static implicit operator GetEmployerMemberResult?(Employer? employer)
+        {
+            if (employer == null)
+                return null;
+
+            return new GetEmployerMemberResult
+            {
+                Email = employer.Email!,
```

Comment:
I don't think you need the nullable flag here if the properties have a default value, and you've already checked to see if Employer is null.